python/ray/worker.py

```diff
@@ -1839,6 +1839,12 @@ def init(redis_address=None,
         logger.info("Detected environment variable 'RAY_USE_XRAY'.")
         use_raylet = True
 
+    if not use_raylet:
+        logger.info("NOTE: Please run Ray with the environment variable "
+                    "'RAY_USE_XRAY=1' to use the latest backend. If you are "
+                    "using Ray on a cluster, 'ray start' must run with this "
+                    "environment variable as well.")
```

Comment:
You could note that this manual configuration is temporary and will be on by default in a future release.